utils/funcexp/func_math.cpp

```diff
@@ -348,6 +378,32 @@ double Func_atan::getDoubleVal(Row& row,
         }
         break;
 
+        case execplan::CalpontSystemCatalog::LONGDOUBLE:
+        {
+            // null value is indicated by isNull
+            long double value = parm[0]->data()->getLongDoubleVal(row, isNull);
+
+            if (isNull || (value < -1.0 || value > 1.0))
```

Comment:
does this conditional contain a typo, perhaps? should it not match line 358 above?

Reply:
Inverse tangent can be less than -1 and greater than 1 if i'm not mistaken

Reply:
−π/2 and +π/2 radians are the limits.

Reply:
You are correct. Typo